#The robot's position information and target information

int32 id

geometry_msgs/Point position

geometry_msgs/Point target

float32 calYaw
float32 radYaw
